distributed/utils_test.py

```diff
@@ -1761,3 +1761,65 @@ async def connect(self, *args, **kwargs):
         return LockedComm(
             comm, self.read_event, self.read_queue, self.write_event, self.write_queue
         )
+
+
+def assert_story(story: list[tuple], expect: list[tuple]) -> None:
```

Comment:
Should this be called `assert_worker_story` since the scheduler still uses a different format?

Reply:
changed it